packages/css/src/components/form/form.template.ts

```diff
@@ -0,0 +1,21 @@
+import { Form } from '@dso-toolkit/sources';
+import { html, nothing, TemplateResult } from 'lit-html';
+
+import { formGroupInputTemplate } from './form-groups/input/form-group-input.template';
+
+export function formTemplate({ mode, formGroups }: Form<TemplateResult>) {
+  return html`
+    <form class="form-${mode}">
+      <fieldset>
+        <legend>Form ${mode}</legend>
+        ${formGroups.map(formGroup => {
```

Comment:
Ik denk dat we hier wat slims gaan doen:

```suggestion
        ${formGroups instanceof TemplateResult ? formGroups : formGroups.map(formGroup => {
```

En dan moet in `form.models.ts` het `Form` aangepast worden, zie volgende comment..

Dan kan die decorator weg en kunnen we in `sources/` gebruik maken van `formTemplate()` alsof het de decorator is! Morgenochtend meteen ff kijken..

Reply:
Die opmerking over de decorator snap ik niet?